hapi/src/api/account.api.js

```diff
@@ -715,7 +723,12 @@ const login = async ({ account, password }) => {
     email_verified: { _eq: true }
   })
 
-  if (!user) {
+  if(user){
```

Comment:
@Edgar097 consider to add this logic if as follow:
`if (user && user.active === "inactive") {`

Reply:
The problem is that if the user is undefined, i mean doesnt exist, the validations "user.active === "inactive" will return an error because there is no ".active" in an undefined value.

Reply:
@leisterfrancisco please check the conflict files

Reply:
@Edgar097 for that reason user is before of user.state, to avoid that

Reply:
What i mean is that it can't be in the same line thats why it is separated, i dont know if that is what you mean

Reply:
@Edgar097  It does, the comparison operator is an **and**, so it means if the first case fail, is not necessary to continue with others comparison.

Reply:
But don't worry, I'll solve it

Reply:
Oh I'm sorry, didn't know that